pint_xarray/tests/test_conversion.py

```diff
@@ -196,11 +196,7 @@ def test_attach_units(self, obj, units):
                 {"a": "K", "b": "hPa", "u": "m"},
                 id="Dataset",
             ),
-            pytest.param(
-                Variable("x", []),
-                {None: "hPa"},
-                id="Variable",
-            ),
+            pytest.param(Variable("x", []), {None: "hPa"}, id="Variable",),
```

Comment:
```suggestion
            pytest.param(Variable("x", []), {None: "hPa"}, id="Variable"),
```

Reply:
I'm actually struggling to get black v20.8 on conda on my machine...

Reply:
that's strange, `conda install -c conda-forge black` should work. Did you try creating a new environment? Sometimes, the environments can break, and it usually is easier to just remove it and create a new one.

Reply:
I can only seem to get v19.0 with `conda install -c conda-forge black`, but I haven't tried creating a new environment yet. Anyway if you want to push formatting changes to this so it can be merged that would be quickest.